### [CVE-2025-4515](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-4515)
![](https://img.shields.io/static/v1?label=Product&message=PrivateGPT&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=%3D%200.6.0%20&color=brighgreen)
![](https://img.shields.io/static/v1?label=Vulnerability&message=Origin%20Validation%20Error&color=brighgreen)
![](https://img.shields.io/static/v1?label=Vulnerability&message=Permissive%20Cross-domain%20Policy%20with%20Untrusted%20Domains&color=brighgreen)

### Description

A vulnerability, which was classified as problematic, was found in Zylon PrivateGPT up to 0.6.2. This affects an unknown part of the file settings.yaml. The manipulation of the argument allow_origins leads to permissive cross-domain policy with untrusted domains. It is possible to initiate the attack remotely. The exploit has been disclosed to the public and may be used. The vendor was contacted early about this disclosure but did not respond in any way.

### POC

#### Reference
- https://gist.github.com/superboy-zjc/2a727cb0c1d468f21a91e0416d006ffe

#### Github
No PoCs found on GitHub currently.

